src/nanovg_mtl.m

```diff
@@ -502,11 +503,10 @@ static int mtlnvg__maxVertCount(const NVGpath* paths, int npaths,
 
   MTLRenderPassDescriptor *descriptor = \
       [MTLRenderPassDescriptor renderPassDescriptor];
-  descriptor.colorAttachments[0].clearColor = s_colorBufferClearColor;
-  descriptor.colorAttachments[0].loadAction = s_colorBufferLoadAction;
+  descriptor.colorAttachments[0].clearColor = mtl->clearColor;
+  descriptor.colorAttachments[0].loadAction = MTLLoadActionClear;
```

Comment:
Always set to `MTLLoadActionClear` would reduce performance. Color shouldn't be cleared when it doesn't need to.